src/utils/deprecated/deprecated.test.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { mount, render } from 'enzyme';
```

Comment:
Using enzyme functions is totally fine, but in general we're moving toward just `jest` and `react-testing-library`

Reply:
I prefer to use `react-testing-library` too, but we don't have it for this project yet. I will create a proposal where we can discuss the migration plan from `enzyme` to `react-testing-library`

Reply:
🤦 Ha, sorry, forgot which repo I was reviewing! I didn't know that OUI didn't have it - something for us to consider adding. Made a stub issue: https://github.com/opensearch-project/oui/issues/654